scripts/update-bcd.ts

```diff
@@ -1211,11 +1239,10 @@ export const update = (
     clearNonExact(options.exactOnly),
     skip("noStatement", ({
       statements,
-      defaultStatements: [simpleStatement],
-      shared: {versionMap},
+      shared: {versionMap, unmodifiedDefaultStatements},
     }) => {
       if (!statements?.length) {
-        if (hasSupportUpdates(versionMap, simpleStatement)) {
+        if (hasSupportUpdates(versionMap, unmodifiedDefaultStatements)) {
```

Comment:
Hmm, wait, going just by the name, isn't this the *original* BCD data (without any modifications) and not the *updated* BCD data that we're about to write back to disk? It needs to be the updated data, but filtered once again to "default statements", meaning excluding things like prefixed statements, flags, etc.

Reply:
Yes that's correct. Seems like I misunderstood the requirement here. We were checking the updated "default statements" before this change, so I'll walk this back.

Reply:
Ok addressed here https://github.com/openwebdocs/mdn-bcd-collector/pull/967/commits/bec996323b1232a0317bcc90fab54c822c4a0308